src/bson.ts

```diff
@@ -21,7 +21,40 @@ import { BSONSymbol } from './symbol';
 import { Timestamp } from './timestamp';
 export { BinaryExtended, BinaryExtendedLegacy, BinarySequence } from './binary';
 export { CodeExtended } from './code';
-export * from './constants';
+export {
```

Comment:
why did this change?

Reply:
This is what introduces the tslib import, exportStar needs some actions to replicate for commonjs.

https://github.com/microsoft/tslib/blob/fcba93d415a6d84914916769ba17871902dd4598/tslib.es6.js#L114